distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -91,6 +91,34 @@ In the ${catalog-ui}, workspaces can be shared between users at different levels
 
 === Searching with ${catalog-ui}
 
+The ${catalog-ui} can create search queries of several types, include <<_text_search_details,text>>, <<_temporal_search_details,temporal>>, <<_spatial_search_details,spatial>>, <<_type_search_details,"type">>, and <<_advanced_search_details,advanced>> searches.
```

Comment:
Remove the "The" here before `${catalog-ui}`, and in every other place where a "The" is in front of a `${catalog-ui}`. 
This will help https://github.com/codice/ddf/pull/1714